### DISCUSSION TITLE
Make a YouTube video tutorial pls!

### DISCUSSION THREAD

**Discussion by noahnoknnmm** [Zero6992/chatGPT-discord-bot#274] [Ideas]
We need a youtube tutorial cuz i understand nothing with text

**Comment by Shabahz** [comment]
This guy did:
https://www.youtube.com/watch?v=Rff54htWgQ4

**Comment by arxifythedev** [comment]
@berkantdu123 this is interesting

**Reply by berkantdu123** [reply]
> @berkantdu123 this is interesting

I believe that this doesn't use assistants update, so would behave like our current one in the discord

**Reply by arxifythedev** [reply]
oh okay. are you still working on that one? do you need any more API keys?

**Comment by Swiftlyfisher** [comment]
a

**Comment by noahnoknnmm** [comment]
What?

Op do 22 feb 2024 om 06:17 schreef Swiftlyfisher ***@***.***>:
